### PR TITLE
【Hackathon 4th No.214】Arm 虚拟硬件上完成 PP-OCR 文本检测模型的部署与优化方案.

### PR DESCRIPTION
**Pull request by txyugood**

### CODE REVIEW

**Comment by Liliyaw** [comment]
Hello 同学, 感谢你报名参加黑客松比赛并选择本题目。
你的RFC文档我正在reivew, 稍后反馈需要调整的问题哈～

File: rfcs/AVH/20230222_ppocr_deploy_for_arm.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Arm 虚拟硬件上完成 PP-OCR 文本检测模型的部署与优化设计文档
+
+|任务名称 | Arm 虚拟硬件上完成 PP-OCR 文本检测模型的部署与优化 | 
+|---|---|
+|提交作者<input type="checkbox" class="rowselector hidden"> | txyugood | 
+|提交时间<input type="checkbox" class="rowselector hidden"> | 2023-2-22 | 
+|版本号 | V1.0 | 
+|依赖飞桨版本<input type="checkbox" class="rowselector hidden"> | develop | 
+|文件名 | 20230222_ppocr_deploy_for_arm.md<br> | 
+
```
Comment (Leenorwich) [inline rfcs/AVH/20230222_ppocr_deploy_for_arm.md:10]: 123

**Review by Liliyaw — APPROVED** [review]
The RFC doc looks good to me. You can start your development with it as the reference. 
BR,
Liliya

**Review by Liliyaw — APPROVED** [review]

**Review by onecatcn — APPROVED** [review]
LGTM

**Pull request merged**